html/modules/custom/ncms_ui/src/Entity/Content/ContentBase.php

```diff
@@ -416,4 +417,34 @@ public function getCommonTaxonomiesService() {
     return \Drupal::service('ncms_tags.common_taxonomies');
   }
 
+  /**
+   * Builds the render array for displaying the current results as a table.
+   */
+  public function getTags() {
+    /** @var \Drupal\ncms_tags\CommonTaxonomyService $common_taxonomies */
+    $common_taxonomies = \Drupal::service('ncms_tags.common_taxonomies');
+    $supported_fields = $common_taxonomies->getCommonTaxonomyFieldNames();
+    $tags = [];
+    $term_storage = \Drupal::entityTypeManager()->getStorage('taxonomy_term');
+    $entity_data = $this->toArray();
+    // Iterating over $supported_fields instead of $entity_data to assure tags
+    // are ordered by vocabulary first. This is not necessarily important, but
+    // make things look more consistent in the backend.
+    foreach ($supported_fields as $field_name) {
+      if (empty($entity_data[$field_name])) {
+        continue;
+      }
+      $field = $entity_data[$field_name];
+      $tids = array_map(function ($value) {
+        return $value['target_id'];
+      }, $field);
+      $terms = $term_storage->loadMultiple($tids);
+      $field_tags = array_map(function (TermInterface $term) {
+        return $term->getName();
+      }, $terms);
+      $tags = array_merge($tags, $field_tags);
+    }
+    return implode(', ', $tags);
```

Comment:
This should return an array of strings and the imploding should be done from where this method is called.